app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldTrackerActivityViewModel.kt

```diff
@@ -198,9 +204,18 @@ class DeviceShieldTrackerActivityViewModel @Inject constructor(
         sendCommand(Command.OpenVpnSettings)
     }
 
+    fun bannerState(): BannerState {
```

Comment:
how do you feel about moving this logic to the `VpnStore`?  Since the `BannerState` is defined in the data layer, we could just return it's value and push the business logic there.

Reply:
Thanks, updated :+1: